<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleCutoffPage</class>
 <widget class="QWidget" name="VehicleCutoffPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
font:24px  &quot;微软雅黑&quot;;
color:black;
border:2px solid white;
background-color:rgb(255,255,153);
}
QPushButton
{
font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 128, 128);border-top: 2px solid white;border-left: 2px solid white;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>611</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>621</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;
	border-radius:8px ;
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>741</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border:1px solid white;
border-radius:15px;
color:white;</string>
   </property>
   <property name="text">
    <string>电制动切除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>481</y>
     <width>741</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border:1px solid white;
border-radius:15px;
color:white;</string>
   </property>
   <property name="text">
    <string>ACM切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CutoffACMBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>490</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetACMBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CutoffEBBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetEBBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
  </widget>
  <widget class="QLabel" name="EBStateMP1lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>电制动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EBStateM1lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>电制动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EBStateM2lbl">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>电制动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EBStateMP2lbl">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>电制动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CabTC1lbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>31</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TC1traingraphKeynoact.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CabTC2lbl">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>140</y>
     <width>31</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TC2traingraphKeyact.bmp);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ACMStateM1lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMStateTC2lbl">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>400</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DirectTC1lbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TC1traingraphDirection.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ACMStateTC1lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMStateM2lbl">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DirectTC2lbl">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/images/images/TC2traingraphDirection.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HMIEBStateMP1lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HMIEBStateM1lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HMIEBStateM2lbl">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HMIEBStateMP2lbl">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HMIACMStateM1lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HMIACMStateM2lbl">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HMIACMStateMP2lbl">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HMIACMStateMP1lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>661</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="Train1Btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="whatsThis">
       <string>1</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24px, &quot;微软雅黑&quot;;color: black;background-color: rgb(255,255,153);border-top: 2px solid  rgb(0, 0, 0);border-left: 2px solid  rgb(0, 0, 0);border-bottom: 2px solid white;border-right: 2px solid white;
</string>
      </property>
      <property name="text">
       <string>09011</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Train3Btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="whatsThis">
       <string>3</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24px, &quot;微软雅黑&quot;;color: black;background-color: rgb(255,255,153);border-top: 2px solid  rgb(0, 0, 0);border-left: 2px solid  rgb(0, 0, 0);border-bottom: 2px solid white;border-right: 2px solid white;
</string>
      </property>
      <property name="text">
       <string>09013</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="Train4Btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="whatsThis">
       <string>4</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24px, &quot;微软雅黑&quot;;color: black;background-color: rgb(255,255,153);border-top: 2px solid  rgb(0, 0, 0);border-left: 2px solid  rgb(0, 0, 0);border-bottom: 2px solid white;border-right: 2px solid white;
</string>
      </property>
      <property name="text">
       <string>09014</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="Train5Btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="whatsThis">
       <string>5</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24px, &quot;微软雅黑&quot;;color: black;background-color: rgb(255,255,153);border-top: 2px solid  rgb(0, 0, 0);border-left: 2px solid  rgb(0, 0, 0);border-bottom: 2px solid white;border-right: 2px solid white;

</string>
      </property>
      <property name="text">
       <string>09015</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="Train6Btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="whatsThis">
       <string>6</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24px, &quot;微软雅黑&quot;;color: black;background-color: rgb(255,255,153);border-top: 2px solid  rgb(0, 0, 0);border-left: 2px solid  rgb(0, 0, 0);border-bottom: 2px solid white;border-right: 2px solid white;
</string>
      </property>
      <property name="text">
       <string>09016</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Train2Btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="whatsThis">
       <string>2</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24px, &quot;微软雅黑&quot;;color: black;background-color: rgb(255,255,153);border-top: 2px solid  rgb(0, 0, 0);border-left: 2px solid  rgb(0, 0, 0);border-bottom: 2px solid white;border-right: 2px solid white;
</string>
      </property>
      <property name="text">
       <string>09012</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>returnBtn</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>CutoffACMBtn</zorder>
  <zorder>ResetACMBtn</zorder>
  <zorder>CutoffEBBtn</zorder>
  <zorder>ResetEBBtn</zorder>
  <zorder>CabTC1lbl</zorder>
  <zorder>CabTC2lbl</zorder>
  <zorder>DirectTC1lbl</zorder>
  <zorder>DirectTC2lbl</zorder>
  <zorder>Train6Btn</zorder>
  <zorder>Train4Btn</zorder>
  <zorder>Train3Btn</zorder>
  <zorder>Train1Btn</zorder>
  <zorder>Train5Btn</zorder>
  <zorder>HMIACMStateMP2lbl</zorder>
  <zorder>HMIACMStateM1lbl</zorder>
  <zorder>HMIEBStateM1lbl</zorder>
  <zorder>ACMStateM1lbl</zorder>
  <zorder>HMIEBStateM2lbl</zorder>
  <zorder>EBStateMP1lbl</zorder>
  <zorder>ACMStateTC2lbl</zorder>
  <zorder>EBStateM1lbl</zorder>
  <zorder>ACMStateM2lbl</zorder>
  <zorder>ACMStateTC1lbl</zorder>
  <zorder>HMIEBStateMP2lbl</zorder>
  <zorder>EBStateM2lbl</zorder>
  <zorder>HMIACMStateMP1lbl</zorder>
  <zorder>HMIACMStateM2lbl</zorder>
  <zorder>EBStateMP2lbl</zorder>
  <zorder>HMIEBStateMP1lbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
